README.md

```diff
@@ -37,51 +37,74 @@ Check out the [🌩 **Nimbus Documentation Hub**](https://experimenter.info) or
 1. Prerequisites
 
     On all platforms:
-    - Install [Docker](https://www.docker.com/)
-    - Install [Node](https://nodejs.org/en/) ^14.0.0
-
+    - Install [Node](https://nodejs.org/en/) ^14.0.0 <17.0 (recommended: 16.9.1)  
```

Comment:
We might want to keep [these consistent](https://github.com/mozilla/experimenter/blob/main/app/Dockerfile#L29)?

Reply:
Ahh that's a good point. I figured this out when running `yarn install` -- it gave me an error that it needed to be less than 17.0. Keeping it consistent with that version works better

Reply:
blah I don't like my wording. thoughts?